src/mscorlib/model.xml

```diff
@@ -3693,21 +3733,26 @@
       <Member Name="#ctor" />
       <Member Name="get_CultureInfo" />
       <Member Name="get_CultureName" />
+      <Member Name="get_CodeBase" />
```

Comment:
aren't we also missing something like:
```xml
<Member MemberType="Property" Name="CodeBase" />
```
for each of these new properties? I'm not sure if that is required.